samples/react/authentication-with-popup/package.json

```
@@ -1,11 +1,12 @@
 {
-  "name": "authentication-with-popup",
-  "version": "0.1.0",
+  "name": "authentication-with-popup-sample",
+  "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "@cognite/sdk": "^2.31.1",
```

```suggestion
    "@cognite/sdk": "^2",
```
does this work? or then it doesn't fetch the local one?

---

The react samples are removed from the workspaces either way, so it doesn't get the local one. However, we should probably just say `^2` because lerna is no longer going to update that. We could in theory have yarn workspaces not include react, and have lerna workspaces include react, but I'll have to see if that breaks something.

---

But are you suggesting we use ^2 in every sample in general?

---

> But are you suggesting we use ^2 in every sample in general?
yes

>The react samples are removed from the workspaces either way
```
  "workspaces": [
    "packages/**",
    "samples/*"
  ],
```
aren't they?

---

Yes, as they are two folders deep. Only one asterisk on `samples/*`